.release/buildspec_sign.yml

```diff
@@ -44,6 +44,21 @@ phases:
       - ls -lah $(dirname `head -1 ./$GIT_COMMIT_ID.manifest`)
   post_build:
     commands:
+      - echo "Copying artifacts to generate a latest binary"
+      - VERSION=`cat ./$GIT_COMMIT_ID.version`
+      - ARTIFACT_DIRECTORY=`cat ./$GIT_COMMIT_ID.artifactdir`
+      - |
+        for platform in windows darwin linux linux-arm64
+        do
+            extension=""
+            if [ "$PLATFORM" = "windows"]; then
+                extension=".exe"
+            fi
+            cp $ARTIFACT_DIRECTORY/copilot-$platform-$VERSION$extension $ARTIFACT_DIRECTORY/copilot-$platform$extension
```

Comment:
Why do we remove the version number here, instead of doing it when we generate those binaries?

Reply:
It was in case the signing Lambda took too long to sign the binaries, Austin said that we sign 9 binaries for the ecs-cli so it should be fine for us as well and moved the logic to the buildspec itself